File: eventbus/app/src/main/java/com/example/eventbus/EmptyActivity.java
package com.example.eventbus;

import android.os.Bundle;
import android.view.View;

import com.example.eventbus.library.BlankBaseLibraryFragment;

import org.greenrobot.eventbus.EventBus;

import androidx.appcompat.app.AppCompatActivity;

public class EmptyActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_empty);
    }

    public void onPostStickyButtonClick(View view) {
        EventBus.getDefault().postSticky(new BlankBaseLibraryFragment.SampleEvent());
        finish();
    }

}

File: eventbus-kotlin/app/src/main/java/com/example/eventbuskotlin/EmptyActivity.kt
package com.example.eventbuskotlin

import android.os.Bundle
import android.support.v7.app.AppCompatActivity
import android.view.View

import org.greenrobot.eventbus.EventBus

class EmptyActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_empty)
    }

    fun onPostStickyButtonClick(view: View) {
        EventBus.getDefault().postSticky(BlankBaseFragment.SampleEvent())
        finish()
    }

}
